lnrpc/lightning.proto

```diff
@@ -755,6 +755,19 @@ message GetTransactionsRequest {
 
     // An optional filter to only include transactions relevant to an account.
     string account = 3;
+
+    /*
+    The index of a transaction that will be used in a query to determine which
+    transaction should be returned in the response, where the index_offset is
```

Comment:
I don't understand the "where the index_offset is excluded" part. If I set it to 0, wouldn't I get all transactions including the one at index 0?

Reply:
Yes, passing 0 should return including the transaction at index 0. I will reword this.